### ISSUE TITLE
Need a simple deployment method that lay down all the ocm pieces

### ISSUE THREAD

**Issue by mikeshng** [open-cluster-management-io/community#36]
Currently, we don’t have an easy way to stand up the entire community edition of open cluster management. Ideally, we should have an installer or an operator that performs the deployment. 

This deployment is about getting all the open pieces to stack up properly and requires efforts from all components and not just from install/mch-operator.

The current deployment method is either via OKD operator hub marketplace or install from component source repo. See https://open-cluster-management.io for more details.

**Comment by mikeshng** [comment]
we are planning to resolve this issue with the `clusteradm` project. For more information: 
see https://github.com/open-cluster-management-io/clusteradm or https://github.com/open-cluster-management/clusteradm

**Comment by juliana-hsu** [comment]
@mikeshng Will clusteradm allow the different components we contribute to CNCF to be installed individually?

**Comment by mikeshng** [comment]
@juliana-hsu yes, it will have functions like `clusteradm enable subscription --cluster cluster1,cluster2 ` that will deploy the appsub addon.

But since its not available yet, maybe I closed this issue prematurely. Reopening to keep track of the progress.

**Comment by mikeshng** [comment]
/assign ycao56 

In the future, when we have policy and app addons then we can extend `clusteradm` to install those addons.

**Comment by mikeshng** [comment]
https://github.com/open-cluster-management-io/clusteradm/pull/46

PR opened for simplify app lifecycle add-on deployment using `clusteradm`

**Comment by mikeshng** [comment]
The PR for app addon deployment using `clusteradm` has now been merged.

**Comment by mikeshng** [comment]
https://github.com/open-cluster-management-io/clusteradm/ now exists with addons support.
